src/config/2019.json

```
@@ -1056,6 +1056,16 @@
       "website": "https://data-seo.com",
       "github": "voltek62"
     },
+    "hakaCode": {
```

Need to be consistent with what you've used in the chapter:

```suggestion
    "HakaCode": {
```